CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Change Log
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+ 
+## [1.2.0] - 2023-11-8
+  
+[Compatible API amendments](https://github.com/oracle/quicksql/issues/23)
+with old API marked as deprecated
+
+The former calls `toDDL` and `toERD`
+```
+   let output = ddl.toDDL(input,opt);
+```
+are encouraged to be replaced with
+
+```
+   let parsed = new parsed(input,opt);
```

Comment:
We committed to not change the API, I can't approve these changes until they're discussed internally

Reply:
@mig8447 if just "deprecated", as long as toDDL/toERD is able to utilize the latest under the hood for the foreseeable future and not dropped, then this is ok. We just can not remove the signature for quite a while. But a new signature should be fine.

Reply:
But the code below looks like toDDL is actually... removed? Which we can not have that happen.

Reply:
Yes @mig8447 lets discuss internally.